app_gui/src/main.rs

```diff
@@ -125,17 +125,19 @@ impl eframe::App for App {
                 }
             });
             ui.separator();
-            ui.heading("Used items");
-            egui::ScrollArea::vertical().show(ui, |ui| {
-                for (i, item) in self.used_items.iter().enumerate() {
-                    ui.dnd_drag_source(
-                        egui::Id::new(item.name.clone()),
-                        self.items.len() + i,
-                        |ui| {
-                            ui.label(&item.name);
-                        },
-                    );
-                }
+            egui::Grid::new("used items title").show(ui, |ui| {
+                ui.heading("Used items");
+                egui::ScrollArea::vertical().show(ui, |ui| {
+                    for (i, item) in self.used_items.iter().enumerate() {
+                        ui.dnd_drag_source(
+                            egui::Id::new(item.name.clone()),
+                            self.items.len() + i,
+                            |ui| {
+                                ui.label(&item.name);
+                            },
+                        );
+                    }
+                });
```

Comment:
Used items appear alongside the heading on my end. How about something like this instead?

```suggestion
                ui.collapsing("Used items", |ui| {
                    egui::ScrollArea::vertical().show(ui, |ui| {
                        for (i, item) in self.used_items.iter().enumerate() {
                            ui.dnd_drag_source(
                                egui::Id::new(item.name.clone()),
                                self.items.len() + i,
                                |ui| {
                                    ui.label(&item.name);
                                },
                            );
                        }
                    })
                })
```